.github/pull_request_template.md

```diff
@@ -1,23 +1,79 @@
 <!--
+A few suggestions about filling out this PR
 
-Replace this comment with a description of what's being changed by this PR.
+1. Use a descriptive title for the PR.
+2. If this pull request is work in progress, create a draft PR instead of prefixing the title with WIP.
+3. Please label this PR at least one of the following labels, depending on the scope of your change:
+- New feature, which adds new behaviors
+- Bug fix
+- Enhancement, which modifies existing behaviors
+- Breaking change
+4. Remove those recommended/optional sections if you don't need them. Only "What does this PR do" and "Checklist" are mandatory.
+5. Submit the pull request: Push your local changes to your forked copy of the repository and submit a pull request (https://help.github.com/articles/using-pull-requests).
+6. Please be patient. We might not be able to review your code as fast as we would like to, but we'll do our best to dedicate to it the attention it deserves. Your effort is much appreciated!
+-->
 
-If this PR should close an issue, please add one of the magic keywords
-(e.g. Fixes) followed by the issue number. For more info see:
-https://help.github.com/articles/closing-issues-using-keywords/
+## What does this PR do?
+<!-- _(Mandatory)_
```

Comment:
Is it ok to just do `Closes #123` here if that issue describes what it does?

Reply:
for the record, I think it is ok to do that

Reply:
Oh sorry, I guess I missed this one 😞 

Because of github's markdown nature, it's possible to close an issue from anywhere. The idea of having a dedicated section is for cleanness/order, but it's totally possible to add that keyword where ever the sender wants.

Reply:
What I meant was, instead of explaining what the PR does, it's probably ok to link to an issue that describes what's to be done.

Reply:
Ah sorry, I did not get it. Yeah, It's totally fine. We can add a more explanatory comment in here